style-guide.adoc

```diff
@@ -658,22 +664,25 @@ Use ordinal numbers as follows:
 
 Don't use commas in numbers.
 In numbers of 5 or more digits, use a nonbreaking space (press option-space bar).
-Breaking up large numbers makes them easier to read, especially in tables.
 A nonbreaking space can’t be misinterpreted as a decimal marker.
+Breaking up large numbers makes them easier to read, especially in tables.
 
 ((&#10004;)) 5000
 
+((&#10004;)) 20 000
+
+((&#10004;)) EUR 3 426 869 were missing from the bank account
```

Comment:
Minor point: How would we write 3 426 869 with a decimal (for cents)? Would it be:

3 426 869.24

Reply:
Yes, exactly. Might look a bit weird, but it's not likely to happen unless we display measurement results.